thirdeye-ui/src/app/components/cron-editor/cron-editor.component.tsx

```diff
@@ -0,0 +1,375 @@
+                                    {clockValue === "pm" ? " 1" : " 0"} -
+                                    {clockValue === "am" ? " 11" : " 12"}
+                                </FormHelperText>
+                            </Grid>
+                            <Grid item>
+                                <TextField
+                                    InputProps={{
+                                        className:
+                                            cronEditorStyle.cronInputField,
+                                    }}
+                                    type="text"
+                                    value={minute}
+                                    variant="outlined"
+                                    onChange={handleMinuteUpdate}
+                                />
+                                <FormHelperText>
+                                    {t("label.minute")}: 0 - 59
+                                </FormHelperText>
+                            </Grid>
+                            <Grid item>
+                                <TextField
+                                    select
+                                    InputProps={{
+                                        className:
+                                            cronEditorStyle.cronInputField,
+                                    }}
+                                    id="interval-select"
+                                    value={clockValue}
+                                    variant="outlined"
+                                    onChange={handleClockTypeChange}
+                                >
+                                    {ClockTimeOptions.map((option) => (
+                                        <MenuItem key={option} value={option}>
+                                            {t(`label.${option}`)}
+                                        </MenuItem>
+                                    ))}
+                                </TextField>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                    <CronHelper
+                        cron={dayTimeCron}
+                        isCronValid={isDayTimeCronValid}
+                    />
+                </>
+            ) : (
+                <>
+                    <Grid container>
+                        <Grid item lg={2} md={4} sm={12} xs={12} />
+                        <Grid item lg={3} md={8} sm={12} xs={12}>
+                            <TextField
+                                fullWidth
+                                error={!isCronValid}
+                                value={cronInternal}
+                                variant="outlined"
+                                onChange={handleCronInputChange}
+                            />
+                        </Grid>
+                    </Grid>
+                    <CronHelper cron={cronInternal} isCronValid={isCronValid} />
```

Comment:
Remove this when this component is refactored to focus on getting user friendly input to convert to cron